models/user.go

```diff
@@ -1617,6 +1620,10 @@ func (opts *SearchUserOptions) toConds() builder.Cond {
 		cond = cond.And(builder.Eq{"is_active": opts.IsActive.IsTrue()})
 	}
 
+	if !opts.HideFromExplorePage.IsNone() {
+		cond = cond.And(builder.Eq{"hide_from_explore_page": opts.HideFromExplorePage.IsTrue()})
+	}
```

Comment:
```suggestion
	if opts.Actor != nil && !opts.Actor.IsAdmin() {
		cond = cond.And(builder.Eq{"hide_from_explore_page": opts.HideFromExplorePage.IsTrue()}.Or(builder.Eq{"id": opts.Actor.ID}))
	} else {
		cond = cond.And(builder.Eq{"hide_from_explore_page": false})
	}
```

Reply:
Added a bit formated, remove useles field in searchOpts